### DISCUSSION TITLE
Logo or attachment upload error

### DISCUSSION THREAD

**Discussion by eguvenc** [uvdesk/community-skeleton#709] [Q&A]
When i try to upload logo or attachment i get an error like : 500 - Internal server error
Is there a upload folder that needs permissions ?

![image](https://github.com/uvdesk/community-skeleton/assets/482833/b3fb2c4c-a864-45f5-9478-333393253c0f)

**Comment by PeopleInside** [comment]
Hi, with PHP version are you using?
With Uvdesk version?

About folder permission you can try to read this topic: https://forums.uvdesk.com/topic/2002/file-permissions-in-the-case-you-have-throbles-installing-uvdesk

I hope as soon UVdesk staff will see that topics will add details if needed.
I hope you will be able to resolve soon.
Also check your error server log if the issue persist.
